### ISSUE TITLE
Broken stroke rendering

### ISSUE THREAD

**Issue by Kiisu-Master** [MewPurPur/GodSVG#319]
Tag with stroke is weirdly cut off.
- This happens randomly when re-rendering, even when the SVG hasn't changed
- Only happens when the tag opacity is below 1
- Happens at resolutions around 100px - 400px
- Happens on both rect and path (Haven't really tested other shapes)
- Happens with any stroke-linejoin setting
- Does not happen when fill is "none"

Reproduce from scratch with rect:
- Set root tag width and height to 150
- Turn on rasterized svg in visual settings
- Add rect
  - make it bigger
  - give stroke width
  - set stroke color to any
- start changing rect opacity

Other example with modified (to show comparison) arrow SVG from icons:
```svg
<svg width="150" height="150" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><path d="M 0.5333 0.7067 l 4.5 7.5 l 4.5 -7.5 z" opacity="0.83" fill="#def" stroke="#def" stroke-width="2" stroke-linejoin="round"/><path d="M 8.56 7.1067 l 4.5 7.5 l 4.5 -7.5 z" fill="#def" stroke="#def" stroke-width="2" stroke-linejoin="round"/></svg>
```
![image](https://github.com/MewPurPur/GodSVG/assets/142301391/55ddce29-fab6-4f81-aba8-302b0415097e)

**Comment by MewPurPur** [comment]
I couldn't pinpoint it to ThorVG, but there are clues that it's some bad combination of ThorVG and Godot. Maybe async stuff not being ready when the texture is drawn sometimes or something, Idk that stuff.

`<svg width="8" height="8" viewBox="0 0 8 8" xmlns="http://www.w3.org/2000/svg"><line x1="4" y1="1" x2="6" y2="3" opacity=".6" stroke="#fac"/><line x1="1" y1="1" x2="3" y2="3" opacity=".6" stroke="#def"/></svg>`

In this SVG, the line that's first gets artifacts as I move around. If I change their order, it's always the one that's first in the code.

**Comment by capnm** [comment]
It was probably a Godot bug. Even if I use a correct XML syntax in v4.2.1, I see this issue. However, I can't reproduce this in v4.3dev2 or github tip.
```gdscript
res://src/ui_parts/display_texture.gd:
...
#svg_text = '<svg viewBox="0 0 8 8" xmlns="http://www.w3.org/2000/svg"><line x1="5" y1="2" x2="6" y2="6" opacity=".4" stroke="#fac" stroke-width="2" stroke-linecap="round"/><line x1="2" y1="2" x2="3" y2="6" opacity=".4" stroke="#def" stroke-width="2" stroke-linecap="round"/></svg>'
	img.load_svg_from_string(svg_text, 10)
	print(svg_text)
	texture = ImageTexture.create_from_image(img)
	position = display_rect.position
	size = display_rect.size

```

```xml
<svg width="8" height="8" viewBox="0 0 8 8" xmlns="http://www.w3.org/2000/svg"/>
??
<line x1="5" y1="2" x2="6" y2="6" opacity=".4" stroke="#fac" stroke-width="2" stroke-linecap="round"/>
<line x1="2" y1="2" x2="3" y2="6" opacity=".4" stroke="#def" stroke-width="2" stroke-linecap="round"/> 
??
```

**Comment by MewPurPur** [comment]
Fixed in Godot 4.2.2.rc2
